plugins/legacy-plugin-chart-event-flow/src/controlPanel.tsx

```diff
@@ -17,10 +17,17 @@
  * under the License.
  */
 import React from 'react';
-import { t, validateNonEmpty, ColumnOption, columnChoices } from '@superset-ui/core';
```

Comment:
Here is the bad import: `ColumnOption` and `columnChoices` should have been imported from `@superset-ui/chart-controls` instead of `@superset-ui/core`.